akirachix/Magezi-Informational
# Feature/products

# Engineers Checklist

This Pull Request updates the Products and Services components to give more information about the products and services offered.Perhaps answering the following questions will help create a good description:

This is to give more information about the services and products offered.
I have done this by adding two components for the products and the services. There is also responsiveness to ensure users can view the site on phones and desktops.
- List of updates for features, enhancements, or bugfixes

Please delete options that are not relevant.


## How can it be tested?
Open the application.
Navigate to the Products or Services section.
Verify that the text in both paragraphs has appropriate margins and wraps correctly.
Test both components across various screen sizes to ensure proper text scaling and layout.


- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets
- [ Jira Link](https://nakatozainamusana.atlassian.net/jira/software/projects/SCRUM/boards/4/backlog?selectedIssue=SCRUM-95)
